.github/workflows/release.yml

```diff
@@ -56,8 +61,8 @@ jobs:
       - name: Image name builder
         id: image_builder
         run: |
-          IMAGE=$(jq -ecr '.tags | map(select(match("${{ env.REGEX_IMAGE }}", "i"))) | first| sub(":.*$";"")' <<< "$DOCKER_METADATA_OUTPUT_JSON")
-          echo "IMAGE=$IMAGE" >> $GITHUB_OUTPUT
+          IMAGE=$(jq -ecr '.tags | map(select(match("${{ env.REGEX_IMAGE }}"; "i"))) | first| sub(":.*$";"")' <<< "$DOCKER_METADATA_OUTPUT_JSON")
+          echo "IMAGE=$IMAGE" | tee $GITHUB_OUTPUT
```

Comment:
This was a fun one. The comma creates a list, which the `match` function of `jq` treats as a list of patterns to match. So we were always matching the "i" character in the "ghcr.io" part of the image tag. Took me a while to figure out why the workflow works without updating `REGEX_IMAGE` even though it should not.